kubernetes/annotations_and_labels.md

```diff
@@ -133,6 +135,26 @@ querying by shared tooling.
 
 ##### Labels
 
+- `cluster.x-k8s.io/cluster-name`
```

Comment:
IIUC it would contain whatever is in `Meta.Name`, and it's just added as label as it's more convenient when filtering. We can maybe add that as a short description.

Reply:
Added label description at the top of this page, alongside other label descriptions.